api/v1alpha1/spannerautoscaler_conversion.go

```diff
@@ -59,7 +59,9 @@ func (src *SpannerAutoscaler) ConvertTo(dstRaw conversion.Hub) error {
 			Max: int(*src.Spec.MaxProcessingUnits),
 		}
 	}
-	scaleConfig.ScaledownStepSize = int(*src.Spec.MaxScaleDownNodes)
+	if src.Spec.MaxScaleDownNodes != nil {
```

Comment:
I believe this direct dereference of `MaxScaleDownNodes` may fail since `MaxScaleDownNodes` is optional, am I correct? 🤔

Reply:
Yes, it should be checked for `nil` before dereferencing. Thanks!